dashboard/src/modules/components/AuthComponent/common-components.jsx

```diff
@@ -88,14 +90,20 @@ export const AuthForm = () => {
       </CardTitle>
       <CardBody>
         <div className="button-wrapper">
-          <Button variant="primary" onClick={() => navigatePage("/login")}>
+          <Button
+            variant="primary"
+            onClick={() => navigatePage("/dashboard/login")}
+          >
             Pbench Credentials
           </Button>
         </div>
         <div className="account-wrapper">
           <div>
             <span>Need an account?</span>
-            <Button variant="link" onClick={() => navigatePage("/signup")}>
+            <Button
+              variant="link"
+              onClick={() => navigatePage("/dashboard/signup")}
```

Comment:
Shouldn't this be referencing `APP_ROUTES.AUTH_SIGNUP`?

Reply:
Yes, updated.